chartfx-chart/src/main/java/de/gsi/chart/plugins/measurements/utils/DataSetSelector.java

```diff
@@ -29,17 +31,32 @@ public DataSetSelector(final ParameterMeasurements plugin) {
         // wrap observable Array List, to prevent resetting the selection model whenever getAllDatasets() is called
         // somewhere in the code.
         allDataSets = plugin.getChart() != null ? FXCollections.observableArrayList(plugin.getChart().getAllDatasets()) : FXCollections.emptyObservableList();
+
+        final Label label = new Label("Selected Dataset: ");
+        GridPane.setConstraints(label, 0, 0);
         dataSetListView = new ListView<>(allDataSets);
+        GridPane.setConstraints(dataSetListView, 1, 0);
         dataSetListView.setOrientation(Orientation.VERTICAL);
         dataSetListView.setPrefSize(-1, DEFAULT_SELECTOR_HEIGHT);
+        dataSetListView.setCellFactory(list -> new DataSetLabel());
+        dataSetListView.setPrefHeight(Math.max(2, allDataSets.size()) * ROW_HEIGHT + 2.0);
         MultipleSelectionModel<DataSet> selModel = dataSetListView.getSelectionModel();
-        if (!allDataSets.isEmpty()) {
-            selModel.select(0);
+        if (requiredNumberOfDataSets == 1) {
+            selModel.setSelectionMode(SelectionMode.SINGLE);
```

Comment:
This doesn't seem to work as intended, I can e.g. select multiple datasets in the dialog for `DataSet1 - value`.

Reply:
fixed: superfluous line in DataSetMeasurements#initialize()...
removed: "SetListView().getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);"